src/services/MemoryService.ts

```diff
@@ -1,4 +1,5 @@
-import { apiClient, tryRequest } from '@/api'
+import { useApiClient } from '../composables/useApiClient'
+const { apiClient, tryRequest } = useApiClient()
```

Comment:
wrong usage, you should not use composables outside of Vue components or Pinia stores.

Reply:
Find a way? I'm asking for it XD
I suggested to avoid `useCookie` and `useJWT` so I can instantiate the client in pure TS (outside Vue logic). 

If not in this way, there is a loop because I would instantiate the client in a store, but other services need to use it. Not possible to do store -> service -> store

Unless I can't see something, in that case do you got any hint?

Reply:
pass the api client instance to stores when initializing the services inside the stores. do not import globally like this